src/util.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from 'path';
+
+export function getTempDir() {
```

Comment:
I've seen this method copy/pasted a few times throughout our actions 😄  This is fine for now, but I opened an issue in `toolkit` so that it's added to one of our packages: https://github.com/actions/toolkit/issues/518